tidb-cloud/serverless-audit-logging.md

```diff
@@ -0,0 +1,230 @@
+```
+
+To create a filter rule that filters ALL EXECUTE events, run the following command:
+
+```shell
+ticloud serverless audit-log filter create --cluster-id <cluster-id> --name <rule-name> --rule '{"users":["%@%"],"filters":[{"classes":["EXECUTE"]]}'
+```
+
+### Update a filter rule
+
+To disable a filter rule, run the following command:
+
+```shell
+ticloud serverless audit-log filter update --cluster-id <cluster-id> --name <rule-name> --enabled=false
+```
+
+To update a filter rule, run the following command:
+
+```shell
+ticloud serverless audit-log filter update --cluster-id <cluster-id> --name <rule-name> --rule '{"users":["%@%"],"filters":[{"classes":["QUERY"],"tables":["test.t"]}]}'
+```
+
+Note that you need to pass the complete `--rule` field when updating.
+
+### Delete a filter rule
+
+To delete a filter rule, run the following command:
+
+```shell
+ticloud serverless audit-log filter delete --cluster-id <cluster-id> --name <rule-name>
+```
+
+## Configure audit logging
+
+### Data redaction
+
+TiDB Cloud Serverless redacts sensitive data in the audit logs by default. For example, the following SQL statement:
+
+```sql 
+INSERT INTO `test`.`users` (`id`, `name`, `password`) VALUES (1, 'Alice', '123456');
+```
+
+It is redacted as follows:
+
+```sql
+INSERT INTO `test`.`users` (`id`, `name`, `password`) VALUES ( ... );
+```
+
+If you want to disable redaction, use the [TiDB Cloud CLI](/tidb-cloud/ticloud-auditlog-config.md).
+
+```shell
+ticloud serverless audit-log config --cluster-id <cluster-id> --unredacted
+```
+
+### Rotation
+
+TiDB Cloud Serverless generates a new audit log file when one of the following conditions is met:
+
+- The audit log file reaches 100 MB.
+- The time interval reaches 1 hour. Note that audit log file generation might be delayed for a few minutes depending on the underlying schedule.
+
+> **Note:**
+>
+> The rotation cannot be configured at present. TiDB Cloud Serverless automatically rotates the audit log files based on the preceding conditions.
+
+## Access audit logging
+
+TiDB Cloud Serverless audit logs are stored as readable text files named `YYYY-MM-DD-<index>.log`.
+
+Currently, audit logs are stored within TiDB Cloudfor 365 days. After this period, logs are automatically deleted.
+
+> **Note:**
+>
+> Support for external storage options (such as AWS S3, Azure Blob Storage, and Google Cloud Storage) will be available in the future.
+
+To view and download audit logs, use the [TiDB Cloud CLI](/tidb-cloud/ticloud-auditlog-download.md):
+
+```shell
+ticloud serverless audit-log download --cluster-id <cluster-id> --output-path <output-path> --start-date <start-date> --end-date <end-date>
+```
+
+- start-date: The start date of the audit log you want to download in the format of `YYYY-MM-DD`, for example `2025-01-01`.
+- end-date: The end date of the audit log you want to download in the format of `YYYY-MM-DD`, for example `2025-01-01`.
+
```

Comment:
Not sure if we also need to use '' to wrap the date, like '2025-01-01' in the above command

Reply:
@shiyuhang In tidb-cloud/ticloud-auditlog-download.md, '' is used in the flag description, so please check which is correct and use it consistently. 

![image](https://github.com/user-attachments/assets/5c686553-cfd4-447d-b99b-0b0d75d127aa)